### ISSUE TITLE
Ingame UI

### ISSUE THREAD

**Issue by Arhowk** [Arhowk/Banjoball#8]
This is the last thing I was working on before I left to go work on abyss w/ tnk

![image](https://cloud.githubusercontent.com/assets/3770591/11616875/4764716a-9c54-11e5-81b8-2944eb0ed381.png)

Basically, the main ability would go in the bubble in the right and all the taunts would be in the bottom. (Mana would also curl around the main ability).

Another thought I had was to do it like a CoD or the SotA UI - put the abil in the same spot it is (except a bit farter to the up-left so theres a noticable gap between the skill adn the edge of the screen), give it some border (maybe even do like a gun name and the "ammo" could be the cooldown remaining) and put the emotes flowing from the top (collapsable by hitting tab or smth)

**Comment by CarpeSwag** [comment]
If we do the first one, we still need a minimap. Additionally, it'd be pretty swag if we also had like 3 more bubbles for each of your allies mana + cooldowns.

**Comment by Arhowk** [comment]
Ooh even better- I could custom render something thats really pretty (e.g. it has all of the guys in big bold font along with their abilities cds)

**Comment by CarpeSwag** [comment]
Whatever you think looks best. If it doesn't come out well in the prototypes we can just redo it later before release.

**Comment by CarpeSwag** [comment]
Was talking to Kingfisher earlier, he requested a couple of things for the UI:
1) a transparent minimap in the bottom center of the screen
2) the scores and time listed like a soccer broadcast in the top left of the screen

Here's an image he linked for reference to both of them:
![image](https://cloud.githubusercontent.com/assets/10646605/11626167/8675a810-9c95-11e5-8a6a-2d872abaf970.png)

**Comment by Arhowk** [comment]
The idea for the minimap is pretty similar than what I suggested, but it
will probably go in the left corner because the right corner will contain
ability cooldown. In addition, instead of red and white dots, they will be
hero pictures (and allies will have skill cooldowns) (if i ever get the
motivation lol)

score should be something easy to do.

On a side note, I tried to extract the fountain texture to use as the
background for the skill but failed miserably. If anyone can find out how
to extract that as a .vmat I'd be eternally grateful. (I think the VRF's
YPoPg decoding works now but I dont know how well it works for animated
materials)

If thats not doable, if we can animate the background somehow that would be
1337 so find animated pictures I can use for it. They should be fading away
from a central circle.

**Comment by Arhowk** [comment]
Looks a tad choppy due to recording but smooth ingame (besides mana, dayne fix)

http://gfycat.com/ObedientKindBe

**Comment by CarpeSwag** [comment]
If you haven't already, add some sort of yellowish-goldish (subtitle color, yo) countdown timer centered on the spell during cd. 

Also, IMO tint the image darker when its on cooldown, too, instead of just grayscaled. Might look a bit swaggier.

I'll do the mana change. Do you have plans to add a small button for kick and sprint? It's a nice UI for returning players, but new players will easily get confused.

**Comment by Arhowk** [comment]
Yeah I was doing it but I totally forgot when I was making the video. See
the strikers forum post for the version with the number

I can color down the image too. I also want to do some kind of glowy thing
but I couldn't find a good gif/Css trick

for newbies I currently have two plans

1) a big "how to play" dialog before the map starts to layout the kick
controls and a bit more

2) a "help" button that just brings up the how to play dialog

**Comment by Arhowk** [comment]
Goalie minimap aura http://thumb9.shutterstock.com/display_pic_with_logo/2604952/264710132/stock-vector-green-bamboo-circle-frame-isolated-on-white-background-264710132.jpg
Ball carrier minimap aura (?) http://cdn2.bigcommerce.com/n-biq04i/ag8fb3/products/7764/images/22203/799C-KW__52703__13989.1384127695.330.500.jpg?c=2
Normal ninimap aura (?) http://cdn.1001freedownloads.com/vector/thumb/119201/4-plait_border_circle.png

**Comment by CarpeSwag** [comment]
Looks gucci!

Btw, I was toying around with your custom UI. I think the mana and ability cooldown should start at 12 o'clock and run clockwise as it's a LOT more intuitive than the current design. I can change it whenever you merge your branch in, (since I was fucking around with your code), so don't worry about it.

**Comment by Arhowk** [comment]
translate: rotateZ(90deg)

**Comment by CarpeSwag** [comment]
Yah but also your masks gotta change too.

**Comment by Arhowk** [comment]
I have Midland this weekend so I'll probably be doing some work at the
nighttime, I'll hope to finish the minimap if possible by this weekend...
If you want an easy task to learn panorama you can do the top scoreboard

**Comment by Arhowk** [comment]
No just rotate the panel containing the masks. I'm not redoing that goddamn
double half circle system

**Comment by CarpeSwag** [comment]
Lol I already had it redone is what I'm saying. Don't worry about it, I'll just upload my changes before we ship.

**Comment by Arhowk** [comment]
The top scoreboard? Or rotating the mask

**Comment by CarpeSwag** [comment]
The whole ability icon, but I can get you both.

**Comment by Arhowk** [comment]
Whatever I'll commit what I have when I get home and you can so whatever.
The next time I'll be working at best is Friday night
